src/lxc/confile_utils.c

```diff
@@ -374,6 +374,28 @@ void lxc_log_configured_netdevs(const struct lxc_conf *conf)
 					  sizeof(bufinet6));
 				TRACE("ipv6 addr: %s", bufinet6);
 			}
+
+			if (netdev->type == LXC_NET_VETH) {
+				lxc_list_for_each_safe(cur, &netdev->priv.veth_attr.ipv4_routes, next) {
+					inet4dev = cur->elem;
+					if (NULL == inet_ntop(AF_INET, &inet4dev->addr, bufinet4, sizeof(bufinet4))) {
+						ERROR("invalid ipv4 veth route");
```

Comment:
The `invalid` should be capitalized

Reply:
@brauner happy to do that, I was following the existing error convention in that function, e.g. https://github.com/lxc/lxc/blob/master/src/lxc/confile_utils.c#L320 but I'll update that instance too to be consistent with the wider project.